src/tests/core.py

```diff
@@ -238,6 +240,19 @@ def test_version(self):
         streams = decoded[7:]
         self.assertEqual(streams, [1, 2, 3])
 
+    def test_insert_method_msgid(self):
+        """Test insert method of helper_sent module with message sending"""
+        fromAddress = 'BM-2cTrmD22fLRrumi3pPLg1ELJ6PdAaTRTdfg'
+        toAddress = 'BM-2cVWtdUzPwF7UNGDrZftWuHWgjdfkj89fdf'
+        message = 'test message'
+        subject = 'test subject'
+        status = 'msgqueued'
+        result = helper_sent.insert(
+            toAddress=toAddress, fromAddress=fromAddress, subject=subject,
+            message=message, status=status,
+        )
+        self.assertNotEqual(result, '')
```

Comment:
It should retrieve the row from the sent table based on `result` and do some check on the values.